examples/plugins/tasks/attractors/visualize_2d_task.py

```diff
@@ -64,9 +65,7 @@ def _default_layout_default(self):
 
     def _models_default(self):
         from model.henon import Henon
-        from model.lorenz import Lorenz
-        from model.rossler import Rossler
-        return [ Henon(), Lorenz(), Rossler() ]
+        return [Henon()]
```

Comment:
Yep, seems like I didn't understand the example correctly. Do you seem the same failure as me?